src/displayPokemon.js

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable no-new */
+const cardPoke = document.querySelector('.card');
+const $grafica = document.querySelector('#grafica');
+const etiquetas = [];
+const datas = [];
+let nroAbility = 0;
+let datosIngresos = {};
+const displayPokemon = (array1) => {
+  // eslint-disable-next-line no-restricted-syntax
+  for (const card of array1) {
+    // eslint-disable-next-line no-undef
+    cardPoke.innerHTML += `
+    <img src="${card.image}" alt="pokemon">
+    <h2>${card.name.toUpperCase()}</h2>
+    `;
+    // eslint-disable-next-line no-restricted-syntax
+    for (const i of card.abilities) {
```

Comment:
It's recommended to give meaningful names to the variables so the code can be understood easier, so it's better to change the variable name "i" for a more descriptive one.